docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,98 @@
 = Glossary of terms related to {ProductName}
+:icons: font
+:source-highlighter: highlightjs
+
+**{ProductName}** +
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+
+**build pipeline customization** +
+The ability to update and manage build pipelines for each component in an application. 
+
+**cluster** +
+A Kubernetes deployment with nodes that run containerized applications and a control plane that manages the nodes.
+
+**commit** +
+A change to one or more files. In {ProductName}, commits that you make to a linked git repository move through the pipeline and automatically get published.
+
+**component** +
+An image that {ProductName} builds from source code in a repository. One or more components that run together form an application.
+
+**Conftest** +
+A utility for testing structured configuration data. Use Conftest to validate container information.
+
+**enterprise contract (EC)** +
+A set of release policies that you apply to your application snapshot. Use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
```

Comment:
It isn't necessarily limited just to SLSA guidelines, these are just an example of the policies that can be used.

Reply:
I'm currently revising the sentence to read, "You can use the EC to prevent..."
I understand that the EC isn't limited to checking SLSA conformance, but surely it's still worth emphasizing the relationship between the two; many EC checks are based on SLSA.